kernel/Dockerfile.config

```diff
@@ -0,0 +1,24 @@
+FROM linuxkit/kernel-compile:1b396c221af673757703258159ddc8539843b02b@sha256:6b32d205bfc6407568324337b707d195d027328dbfec554428ea93e7b0a8299b AS kernel-build
+
+ARG KERNEL_VERSION
+ARG KERNEL_SERIES
+
```

Comment:
But this would still compile the kernel. We just want to run `oldconfig` to update the config...or do you suggest the person updating the kernel just to hit ^C once that is done? I personally would prefer a single `make` target, which I run after updating kernel/patches to get the updated config file.